crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -51,31 +52,40 @@ impl From<&StarknetContractArtifacts> for ContractArtifacts {
 }
 
 pub struct CairoHintProcessor<'a> {
-    pub original_cairo_hint_processor: OriginalCairoHintProcessor<'a>,
+    pub original_cairo_hint_processor: SyscallHintProcessor<'a>,
```

Comment:
I guess it uses the hint processor from the compiler under the hood when it comes to non-starknet hints, am I right?

I also wonder if it we should rename it to `original_blockifier_hint_processor` or sth since it supports storage and stuff

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/589/commits/67a7aba7cafe7a66643871b83cabfab397e415e9